runtime/bin/analyze_snapshot.cc

```diff
@@ -169,24 +161,14 @@ int RunAnalyzer(int argc, char** argv) {
   const char* script_name = nullptr;
   script_name = inputs.GetArgument(0);
 
-  // Dart_LoadELF will crash on nonexistent file non-gracefully
-  // even though it should return `nullptr`.
-  File* const file = File::Open(/*namespc=*/nullptr, script_name, File::kRead);
```

Comment:
Actually probably want to add this back.  It doesn't *crash* anymore, but it does give a nonsense error message about snapshot being invalid, instead of saying the file doesn't exist.